### ISSUE TITLE
208 rule is maybe too strict (`mode: preserve` is usable only on very few ansible modules)

### ISSUE THREAD

**Issue by fauust** [ansible/ansible-lint#1000]
##### Summary

There is an error in ansible documentation and `mode: preserve` can only be used on very few modules.
https://github.com/ansible/ansible/issues/56839

So there might be something to do for rule 208 and the documentation should maybe propose another solution that using `mode: preserve`:
https://ansible-lint.readthedocs.io/en/latest/default_rules.html#file-permissions-not-mentioned

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console
❯ ansible --version
ansible 2.9.12
  config file = None
  configured module search path = ['/home/faust/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/faust/.local/lib/python3.7/site-packages/ansible
  executable location = /home/faust/.local/bin/ansible
  python version = 3.7.3 (default, Jul 25 2020, 13:03:44) [GCC 8.3.0]
❯ ansible-lint --version
ansible-lint 4.3.1
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

```console
❯ cat /etc/debian_version
10.5
❯ uname -a
Linux laptop 4.19.0-9-amd64 #1 SMP Debian 4.19.118-2+deb10u1 (2020-06-07) x86_64 GNU/Linux
```

##### STEPS TO REPRODUCE

Ansible code:

```yaml
- name: setup default journal retention
  lineinfile:
    path: /etc/systemd/journald.conf
    regexp: "^#SystemMaxUse="
    line: SystemMaxUse=200M
```

Ansible-lint error:

```console
[208] File permissions not mentioned
roles/common/tasks/main.yml:29
Task/Handler: setup default journal retention
```

Unless it is not considered as a good practice, for `lineinfile` module, I don't want to specify `mode` because ansible should not modify the file mode IMO.

**Comment by ssbarnea** [comment]
v4.3.2 improves E208, tag already created.
